### ISSUE TITLE
Unable to copy from table when the table includes a column group

### ISSUE THREAD

**Issue by jerrens** [olifolkerd/tabulator#2452]
The copying feature (click in any row, then press CTRL+C) works on Tabulator without any problem until I add a column group, then when trying to copy the data, the clipboard is never updated.

In Chrome, I see the following error:
```
tabulator.min.js:6 Uncaught TypeError: t.getFieldValue is not a function
    at tabulator.min.js:6
    at Array.forEach (<anonymous>)
    at tabulator.min.js:6
    at Array.forEach (<anonymous>)
    at l (tabulator.min.js:6)
    at b.generateHTML (tabulator.min.js:6)
    at b.buildOutput (tabulator.min.js:6)
    at b.active [as copySelector] (tabulator.min.js:6)
    at b.generateContent (tabulator.min.js:6)
    at HTMLDivElement.<anonymous> (tabulator.min.js:6)
(anonymous) @ tabulator.min.js:6
(anonymous) @ tabulator.min.js:6
l @ tabulator.min.js:6
b.generateHTML @ tabulator.min.js:6
b.buildOutput @ tabulator.min.js:6
active @ tabulator.min.js:6
b.generateContent @ tabulator.min.js:6
(anonymous) @ tabulator.min.js:6
b.copy @ tabulator.min.js:6
copyToClipboard @ tabulator.min.js:9
F.checkBinding @ tabulator.min.js:9
(anonymous) @ tabulator.min.js:9
keyupBinding @ tabulator.min.js:9
```

I've made an demo at to help troubleshoot: 
https://codepen.io/jerren_s/pen/qBBrQMb

Steps to recreate:
1. Click on any row in the table (to set focus)
2. Press CTRL+C
3. Attempt to paste the clipboard contents into any other application
4. Back in the CodePen, Click the "Add Group Column" button
5. Repeat steps 1-3
6. Note that the pasted content is from the original copy and does not contain the Column Group
- The browser console should also be showing the above error
- If you clear the clipboard after Step 3, then in Step 5, nothing will be on the clipboard

**Comment by olifolkerd** [comment]
Hey @jerrens 

Thanks for the headsup, i will make sure that is fixed in the next release.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Hey @jerrens 

I have pushed a fix for this to the 4.5 branch which will be released in a few weeks.

Cheers

Oli :)
